From d7338bf59407efd5d5147cc8ab081c4739256483 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 1 Aug 2017 15:23:06 +0800
Subject: [PATCH] usb: dwc_otg_310: support battery charger detect for rk3288w

rk3288 and rk3288w use different compatible string, so add
"rockchip,rk3288w" to support rk3288w.

Change-Id: I828f09d26fea30921159f24ce9bdffd286df777a
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/usb/dwc_otg_310/usbdev_bc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/usb/dwc_otg_310/usbdev_bc.c b/drivers/usb/dwc_otg_310/usbdev_bc.c
index 79a08a0a44f9..748b9ee4a454 100755
--- a/drivers/usb/dwc_otg_310/usbdev_bc.c
+++ b/drivers/usb/dwc_otg_310/usbdev_bc.c
@@ -41,7 +41,8 @@ static inline void *get_grf_base(struct device_node *np)
 
 	if (of_machine_is_compatible("rockchip,rk3188"))
 		grf_base -= 0xac;
-	else if (of_machine_is_compatible("rockchip,rk3288"))
+	else if (of_machine_is_compatible("rockchip,rk3288") ||
+		 of_machine_is_compatible("rockchip,rk3288w"))
 		grf_base -= 0x284;
 
 	return grf_base;
-- 
2.35.3

